From 1a6cd019c1ab62ca0dc23bbc6b033df3f15850a5 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@freescale.com>
Date: Wed, 29 Apr 2015 10:29:27 -0500
Subject: [PATCH] MLK-10782-4 ARM:imx6qdl:dts:Add LDB_DI_CLK parent to device
 tree.

Add the LDB clock parents to the device tree.

Signed-off-by: Fabio Estevam <fabio.estevam@freescale.com>
Signed-off-by: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@freescale.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi | 2 ++
 arch/arm/boot/dts/imx6qdl-sabresd.dtsi   | 5 +++++
 2 files changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index aa261aa..1ad504e 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -286,6 +286,8 @@
 	assigned-clock-parents = <&clks IMX6QDL_CLK_LVDS2_IN>,
 				<&clks IMX6QDL_PLL4_BYPASS_SRC>;
 	assigned-clock-rates = <0>, <0>, <24576000>;
+	fsl,ldb-di0-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
+	fsl,ldb-di1-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
 };
 
 &ecspi1 {
diff --git a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
index 9cf7652..33e7200 100644
--- a/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabresd.dtsi
@@ -254,6 +254,11 @@
 	soc-supply = <&sw1c_reg>;
 };
 
+&clks {
+	fsl,ldb-di0-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
+	fsl,ldb-di1-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
+};
+
 &ecspi1 {
 	fsl,spi-num-chipselects = <1>;
 	cs-gpios = <&gpio4 9 0>;
-- 
1.8.0

